application_deploy/demo_app_def.properties

```diff
@@ -0,0 +1,38 @@
+###
+# Pulsar Client Applications
+# 
+# Format:
+#   client_app_id=<prog_language>|'client_app'|<app_path_identifier>|<class_name>|<app_cmd_parameters>
+#
+# Fields: 
+#   - <prog_language> : programming language that the client application is written with| e.g. Java or Python
+#   - 'client_app' : literal as is
+#   - <app_path_identifier> : releative path under 'application_code" folder| e.g. application_code/client_apps/java/[app_path_identifier]
+#   - <class_name> : java class name without package (only relevant with Java apps)| e.g. PulsarConsumerFullCfg 
+#                    package name is standard as 'com.example.pulsarworkshop'
+#   - <app_cmd_parameters> : space separated CLI input paramters of the specified application
+# --------------------------------------------------------------------------------
+
+# A Pulsar message producer using Pulsar native client API
+nativeapi_producer_full_config=java|client_app|nativeapi_producer_full_config|PulsarProducerFullCfg|-numMsg 10 -topic public/default/t1 -connFile /Users/yabinmeng/MyFolder/Yabin.Work/PSA.Vanguard/pulsar_workshop/scenarios/demo-scenario-1/client.conf -srcWrkldFile /Users/yabinmeng/Temp/raw_data_iot_telemetry.csv
```

Comment:
"PSA.Vanguard"  is shown in the -connfile parameter, is that Ok?  More generic name/directory since this is an example.